src/applications/static-pages/analytics/addQaSectionListeners.js

```diff
@@ -0,0 +1,52 @@
+import recordEvent from 'platform/monitoring/record-event';
+import { isAnchor } from './utilities';
+
+const selectors = {
+  template: '[data-template="paragraphs/q_a_section"]',
+  faq: '[data-template="paragraphs/q_a"]',
+};
+
+function attachDataToAnchorTags() {
+  const faqs = document.querySelectorAll(selectors.faq);
+
+  // Cycle through each FAQ, binding the FAQ data to each anchor
+  // tag generated from the WYSIWYG content.
+  [...faqs].forEach(faq => {
+    const anchors = faq.querySelectorAll('a');
+
+    [...anchors].forEach(anchor => {
+      anchor.dataset.faqSection = faq.dataset.analyticsFaqSection; // eslint-disable-line no-param-reassign
+      anchor.dataset.faqText = faq.dataset.analyticsFaqText; // eslint-disable-line no-param-reassign
+    });
+  });
+}
```

Comment:
Could this work, letting you avoid the ESLint disabling?

```js
    [...anchors].map(anchor => ({
      ...anchor,
      dataset: {
        faqSection: faq.dataset.analyticsFaqSection,
        faqText: faq.dataset.analyticsFaqText,
      },
    }))
```

Reply:
I can get past disabling the linter by just destructuring the `dataset` property. Just pushed the commit.